sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -91,8 +91,8 @@ public TelemetryItem(string name, TelemetryItem telemetryItem, ActivitySpanId ac
             }
         }
 
-        public TelemetryItem (LogRecord logRecord, AzureMonitorResource? resource, string instrumentationKey) :
-            this(logRecord.Exception != null ? "Exception" : "Message", FormatUtcTimestamp(logRecord.Timestamp))
+        public TelemetryItem (string name, LogRecord logRecord, AzureMonitorResource? resource, string instrumentationKey) :
```

Comment:
Should we rely on enum with all telemetry types instead of string name here?

Reply:
I removed the inline strings that were in this class. The new code is in the `LogsHelper` class.

I think what you're proposing may be a larger change. We need to manually set both the TelemetryItem name and the BaseData name. Enum may not be the best choice because it's a pair of values, but I could introduce some string constants. 

I suggest we explore this improvement outside of this current PR.